canonical/canonical.com
# Remove travel locations from the FAQ

## Done
Remove travel locations from the FAQ

## QA

- Check the code changes match the comment in the [copy doc](https://docs.google.com/document/d/12_O_QtPqfv9p7kaDFdiP8Xq-kY5xa1XB-C-CRbXjDew/edit)
